R/translate-sql-window.r

```diff
@@ -194,6 +197,14 @@ win_current_frame <- function() sql_context$frame
 # Not exported, because you shouldn't need it
 sql_current_con <- function() sql_context$con
 
+# Functions to manage information for special cases
+set_current_context <- function(context) {
+  old <- sql_context$context
+  sql_context$context <- context
+  invisible(old)
+}
+
+sql_current_context <- function() sql_context$context
```

Comment:
Ok, the only other `sql_current_` function I could find is `sql_current_con` and is not exported/documented: 

https://github.com/tidyverse/dbplyr/blob/2c93c9d37cf56d7dc331df02699739f49064de50/R/translate-sql-window.r#L194-L195